pom.xml

```diff
@@ -11,10 +11,10 @@
         <relativePath /> <!-- lookup parent from repository -->
     </parent>
     <groupId>edu.ucsb.cs156</groupId>
-    <artifactId>example</artifactId>
+    <artifactId>happiercows</artifactId>
     <version>1.1.0</version>
-    <name>example</name>
-    <description>Spring Boot + React Example</description>
+    <name>happiercows</name>
```

Comment:
Also here too

Reply:
have we got approval for this from the professor?